oci/private/pull.bzl

```diff
@@ -91,7 +91,7 @@ def _get_auth(rctx, state, registry):
                 elif "auth" in auth_val:
                     # base64 encoded plaintext username and password
                     raw_auth = auth_val["auth"]
-                    (login, password) = base64.decode(raw_auth).split(":")
+                    (login, password) = base64.decode(raw_auth).split(":", 1)
```

Comment:
```suggestion
                    login, _, password = base64.decode(raw_auth).partition(":")
```
will avoid unpacking errors even if a colon is missing from the string.

Reply:
I am not sure if that's what we want here. we except the basic auth string to contain `:`.

Reply:
True, but getting a "not enough values to unpack" error isn't user-friendly. Maybe:
```suggestion
                    login, sep, password = base64.decode(raw_auth).partition(":")
                    if not sep:
                        fail("Auth string must be in form username:password")
```

Reply:
that would be a corrupted auth file in this case no?

Reply:
I think a proper error message would be nice. I was confused at first when I got the `too many values to unpack`.